lib/ansible/plugins/filter/aws_kms.py

```diff
@@ -0,0 +1,92 @@
+"""
+(c) 2018, Archie Gunasekara <contact@achinthagunasekara.com>.
+Module to handle encrypting and decrypting of items with AWS KMS.
```

Comment:
GPLv3 header needs to be added

Reply:
`# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)`

Reply:
GPLv3 header added